src/deadline/houdini_adaptor/HoudiniAdaptor/adaptor.py

```diff
@@ -200,26 +199,29 @@ def _get_regex_callbacks(self) -> list[RegexCallback]:
         if not self._regex_callbacks:
             callback_list = []
 
-            _houdini_license_error = "RuntimeError: Error encountered when initializing Houdini"
-
             completed_regexes = [re.compile(".*Finished Rendering.*")]
             progress_regexes = [re.compile(".*ALF_PROGRESS ([0-9]+)%.*")]
-            error_regexes = [re.compile(".*Error: .*|.*\\[Error\\].*", re.IGNORECASE)]
+            license_regexes = [
+                # generic runtime error
```

Comment:
Should this be under the license error regex list if it's a generic error?

Reply:
I went back and forth on this a few times. It was previously just a license error, so I'm not changing functionality here.

Putting it with the generic errors would then make the "strict error handling" apply to it which I'm not sure we want.

Reply:
Sounds good.